### PR TITLE
[country-list] Updated country-list types to work with version 2.1

### PR DESCRIPTION
**Pull request by Kimahriman**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/fannarsh/country-list/commit/ef4633adc993834fea4ba11df7bc75f627df1b0f https://github.com/fannarsh/country-list/commit/a0102f2839011665abcd02224eac1e68a9def5f4
- [x] Increase the version number in the header if appropriate.
- [x] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

### CODE REVIEW

**Review by iRoachie — CHANGES_REQUESTED** [review]
Follow these steps when updating to a major version.

https://github.com/DefinitelyTyped/DefinitelyTyped#i-want-to-update-a-package-to-a-new-major-version

**Review by iRoachie — APPROVED** [review]
Thanks!

File: types/country-list/index.d.ts [hunk 1]
```diff
@@ -1,41 +1,45 @@
...
-
-  /**
-   * Returns an array of all country information, in the same format as it gets imported.
-   */
-  getData(): Array<{ code: string, name: string }>;
-};
+/**
+ * Expects an array of code and country name pairs to add to the list. Doesn't return anything.
+ */
+export function overwrite(countries: Array<{ code: string, name: string}>): undefined;
```
Comment (uniqueiniquity) [inline types/country-list/index.d.ts]: Please make the return type of `overwrite` be `void`.

**Review by uniqueiniquity — CHANGES_REQUESTED** [review]

File: types/country-list/index.d.ts (hunk 1 above)
Comment (Kimahriman) [reply types/country-list/index.d.ts]: Whoops

**Review by uniqueiniquity — APPROVED** [review]

**Pull request merged**